File: aziz/Angular/src/app/employe.ts
import { Formation } from './formation';
import { Binary } from '@angular/compiler';

export class Employe {
    id_emp: number;
    nom: string;
    prenom: string;
    Email: string;
    cin: string;
    adress:string; 
    image: string;
    type:string;
    departement:string; 
    specialite:string; 
    date_de_naissance:Date;
    forma:boolean;
    formation:Formation;

    constructor(){}
   // virtual public List<Formation> formations { get; set; }

  
}

File: aziz/Pfe-backend/Domain/Entities/Employe.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{[Table("Employe")]
    public class Employe
    {
        [Key]
        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int? id_emp { get; set; }
        
        public string prenom { get; set; }
        public string nom { get; set; }
        public string Email { get; set; }
       // public byte[] photo { get; set; }
        public int cin { get; set; }
        public string adress { get; set; }
        //[Column(TypeName = "image")]
        public string image { get; set; }
        public Type type { get; set; }//BIATOS Ense ...
        public string departement { get; set; } // dsi
        public string specialite { get; set; }//dev
        public int forma { get; set; }
        public DateTime date_de_naissance { get; set; }
        public string Password { get; set; }
        virtual public Formation formation { get; set; }

      
    }
}
